exchanges/coinbasepro/coinbasepro.go

```diff
@@ -742,6 +748,9 @@ func (c *CoinbasePro) SendAuthenticatedHTTPRequest(ep exchange.URL, method, path
 		Verbose:       c.Verbose,
 		HTTPDebugging: c.HTTPDebugging,
 		HTTPRecording: c.HTTPRecording,
+	}
+	return c.SendPayload(ctx, request.Unset, func() (*request.Item, error) {
+		return item, nil
```

Comment:
This function also uses timestamp based signature/headers. I would advise inlining. Coinbasepro has rate limits